test/boost/service_level_controller_test.cc

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2021-present ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#include <boost/test/unit_test.hpp>
+#include <stdlib.h>
+#include <iostream>
+
+#include "seastarx.hh"
+#include <seastar/testing/test_case.hh>
+#include <seastar/testing/thread_test_case.hh>
+#include <seastar/core/future-util.hh>
+#include <algorithm>
+#include "service/qos/service_level_controller.hh"
+#include "service/qos/qos_configuration_change_subscriber.hh"
+#include "auth/service.hh"
+
+// helper type for the visitor #4
```

Comment:
What is `#4`?

You can use the overloaded_functor helper we have.

Reply:
> What is `#4`?

A copy paste error :relaxed: 

> You can use the overloaded_functor helper we have.


I will do that

Reply:
done